### PR TITLE
Check for empty $tag in add_prefix()

### PR DESCRIPTION
**Pull request by Crizz0**
Give warning, when a user does not fill out the input field for `Create a new topic prefix:`

- [x] Code change
- [ ] Tests

### CODE REVIEW

File: controller/admin_controller.php [hunk 1]
```diff
@@ -154,8 +154,15 @@ public function add_prefix()
 			}
 
 			$tag = $this->request->variable('prefix_tag', '', true);
-			$prefix = $this->manager->add_prefix($tag, $this->forum_id);
-			$this->log($prefix['prefix_tag'], 'ACP_LOG_PREFIX_ADDED');
+			if ($tag != '')
```
Comment (iMattPro) [inline controller/admin_controller.php]: I think I'd prefer you flip this?

```php
if ($tag === '')
{
    $this->trigger_message('TOPIC_PREFIXES_INPUT_EMPTY', E_USER_WARNING);
}

$prefix = $this->manager->add_prefix($tag, $this->forum_id);
$this->log($prefix['prefix_tag'], 'ACP_LOG_PREFIX_ADDED');
```

Comment (iMattPro) [reply controller/admin_controller.php]: Also should update the Add prefix test to test this with an empty string

Comment (Crizz0) [reply controller/admin_controller.php]: `empty()` will check if `$tag` exists, too, which it always will and afaik be `true` if the user tries to use "0" as prefix, or not?

Comment (iMattPro) [reply controller/admin_controller.php]: I edited my original comment because of that

**Comment by iMattPro** [comment]
This is being fixed now in #27
